airgun/entities/contentview.py

```diff
@@ -241,10 +241,8 @@ def remove_version(self, entity_name, version_name, completely=True,
         view.flash.dismiss()
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         result = view.versions.search(version_name)
-        if completely and not result:
-            return
-        view.versions.table.row(
-            version=version_name)['Status'].widget.wait_for_result()
+        if completely and result[0]['Version'] != version_name:
+            return view.versions.table.row(version=version_name)['Status'].widget.wait_for_result()
```

Comment:
Why this change of logic?

Old version was: if `completely == True` and "row is removed from table", then return; otherwise, call `wait_for_result()` on widget.
New version is: if `completely == True` and "row is removed from table", call `wait_for_result()` on widget (widget that is not here anymore); otherwise, do nothin.